CRM/Contact/Form/Task/Delete.php

```diff
@@ -228,15 +212,7 @@ public function postProcess() {
     }
     if ($deleted) {
       $title = ts('Deleted');
-      if ($this->_restore) {
-        $title = ts('Restored');
-        $status = ts('%1 has been restored from the trash.', [
```

Comment:
this seems wrong for multiple contacts but should be no worse for this PR (it might be an empty name rather than a random one but not sure that is worse)

Reply:
It works because of the 'plural' and 'count' params to ts(). You don't see it that much - came from Drupal I think.